src/digitalmarketplace/components/alert/_alert.scss

```diff
@@ -1,10 +1,9 @@
-@import "../../../govuk-frontend/tools/all";
-@import "../../../govuk-frontend/helpers/all";
+@import "../../../govuk-frontend/govuk/tools/all";
+@import "../../../govuk-frontend/govuk/helpers/all";
 
 @include govuk-exports("digitalmarketplace/component/alert") {
 
     .dm-alert {
-        @include govuk-focusable;
```

Comment:
Do we need to do anything to replace `govuk-focusable`?

Reply:
> Thinking about it, I think we will need to merge these changes to a long-running branch and not than the default one, so if we can put as many of the changes as possible into a PR that we can merge into main that would be awesome.
> 
> I've already split the `npm audit` bumps into a separate PR at #150

I think the rest of this is now govuk-v3 dependent. I can rip out all the path changes so this is more easily merged with #151 though.

Potentially could also push the `data-module="govuk-button"` change to main, though that at least has potential to do something weird.